### PR TITLE
Enhancement::ModalDrawer: Option added for disabling the parent scrolling page

### PR DESCRIPTION
**Pull request by saiprasaddevare**
Enhancement :: modal_drawer : added new functionality for disabling the background / parent scroll while modal is open

### CODE REVIEW

File: dist/angular-modal-drawer.js [hunk 1]
```diff
@@ -44,7 +44,11 @@
             if(initOptions.backdropClass){
               backdropElem.classList.add(initOptions.backdropClass);
             }
-          }          
+          }
+          if(initOptions.disableParentScroll){
```
Comment (vadasambar) [inline dist/angular-modal-drawer.js]: Can we have a better name for this ?

File: dist/angular-modal-drawer.css [hunk 2]
```diff
@@ -27,8 +27,12 @@
   position: fixed;
 }
 
+.modalParentDisableScroll{
```
Comment (vadasambar) [inline dist/angular-modal-drawer.css]: Can we have a better name for this ?

File: dist/angular-modal-drawer.js [hunk 3]
```diff
@@ -70,10 +74,15 @@
             }
           }
 
+          if(initOptions.disableParentScroll){
```
Comment (vadasambar) [inline dist/angular-modal-drawer.js]: Can we DRY mutiple if statements here ? (low priority)

File: dist/angular-modal-drawer.css (hunk 2 above)
Comment (saiprasaddevare) [reply dist/angular-modal-drawer.css]: ok

File: dist/angular-modal-drawer.js (hunk 1 above)
Comment (saiprasaddevare) [reply dist/angular-modal-drawer.js]: ok

File: dist/angular-modal-drawer.js (hunk 3 above)
Comment (saiprasaddevare) [reply dist/angular-modal-drawer.js]: yes, will look over it

**Pull request merged**